rust/src/lib/query_apply/route.rs

```diff
@@ -213,3 +213,35 @@ impl RouteEntry {
         }
     }
 }
+
+impl Routes {
+    pub(crate) fn apply_ignored_ifaces(&mut self, ifaces: &Interfaces) {
+        let ignored_ifaces: HashSet<&str> = ifaces
+            .kernel_ifaces
+            .iter()
+            .filter_map(|(name, iface)| {
+                if iface.is_ignore() {
+                    Some(name.as_str())
+                } else {
+                    None
+                }
+            })
+            .collect();
+
+        for rts in [self.running.as_mut(), self.config.as_mut()]
```

Comment:
`routes.running` is ignored in apply, is there any point in marking them with `state: ignore`?

Reply:
In case in the future we will use running config in follow up processing. If we don't do it here, it will be very hard to debug.